src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_skadi.cpp

```diff
@@ -28,11 +28,14 @@ enum Misc
     // TEXTS
     SAY_AGGRO                           = 0,
     SAY_KILL                            = 1,
-    EMOTE_RANGE                         = 2,
     SAY_DEATH                           = 3,
     SAY_DRAKE_DEATH                     = 5,
     SAY_DRAKE_BREATH                    = 6,
 
+    // EMOTES
+    EMOTE_DEEP_BREATH                   = 0,
+    EMOTE_RANGE                         = 1,
```

Comment:
This will be wrong... as 0 would point to SAY_AGGRO and 1 to SAY_KILL. Double check in creature_template for the correct values

Reply:
They come from Grauf, not from Skadi - Grauf's template only has two entries, for each of these emotes.

Reply:
ah ok